### ISSUE TITLE
Remove underscore

### ISSUE THREAD

**Issue by simplenotezy** [EvanAgee/vuejs-wordpress-theme-starter#99]
I noticed you're pulling in underscore, but since you're using babel, you can write ES6.

For instance:

    getPosts(limit, cb) {
      if (_.isEmpty(limit)) {
        let limit = 5;
      }

      axios
        .get(SETTINGS.API_BASE_PATH + "posts?per_page=" + limit)
        .then(response => {
          cb(response.data);
        })
        .catch(e => {
          cb(e);
        });
    }

Could simply be:

    getPosts(limit = 5, cb) {
      axios
        .get(SETTINGS.API_BASE_PATH + "posts?per_page=" + limit)
        .then({data} => {
          cb(data);
        })
        .catch(e => {
          cb(e);
        });
    }

It would reduce filesize ;)

**Comment by simplenotezy** [comment]
Speaking of which; I don't see where / how loadash is being included - not present in the package.json or webpack file.

**Comment by simplenotezy** [comment]
Never mind; it was in package.json =)

**Comment by EvanAgee** [comment]
@simplenotezy yes this is on my list but I haven't had time yet. Interested in creating a PR to remove the dependency?

**Comment by simplenotezy** [comment]
Yes, I'll have a look at it @EvanAgee

**Comment by simplenotezy** [comment]
Just to have an idea of how much lodash takes:

![image](https://user-images.githubusercontent.com/3786627/73960917-ff01d700-490b-11ea-811d-3b13878e05cd.png)

**Comment by EvanAgee** [comment]
Yes, it's a big boy. I'll implement the tree shaking so at least it isn't larger than necessary.

**Comment by simplenotezy** [comment]
It's used sooo little it shouldn't take long time just to completely remove
